Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/30 Configure Simple Items.md

> Use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings (**formData** field names) and objects (item configurations). Use a string if the default item configuration suits you. Otherwise, declare an object that configures the simple item. Specify the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) and other properties in it. In the example below, we configure the `HireDate` simple item:

Reviewer: I wouldn't use the words "simple item" to much at this point. You never explained what "simple" means, or what makes them "simple", and what other kinds are available. Is there any value at this point to refer to them as "simple items"? 

```suggestion
Use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings (**formData** field names) and objects (item configurations). Use a string if the default item configuration suits you. Otherwise, declare an object that configures an item. Specify the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) and other properties in it. In the example below, we configure the `HireDate` item:
```

Author: Hello. Simple Item is the most used type of Form items. As stated above, it consists of a label and an editor. https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/

It is important to show that users can specify strings in the items[] array only for simple items.

Reviewer: "Simple" is just a weird name. If there's a simple item, there should also be an "advanced" item, that's better somehow. I'm not saying you didn't give a formal definition of what simple is - a pair of label and editor. But the name simple implies that there might be a better label-editor pair item, an "advanced" label-editor pair that can probably do more than "simple" items. (Also, maybe you defined it and maybe not... Does it really matter? Maybe the reader skipped that sentence. That's why I like it more when we avoid weird internal terminology.)

If it was called an "editor" item, then fine. But when you repeat the word "simple", I always ask myself a question of why I need to use something "simple" (for dummies?). When will I get to use the real stuff? :-) 

You also say this: "It is important to show that users can specify strings in the items[] array only for simple items." But you never really say this in the document, right? You only use the word "simple" when you talk about objects, not strings. 

Anyway, here's another suggestion. Hope it works :-) 

```suggestion
Use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings (**formData** field names) and objects (item configurations). Use a string to create a label-editor pair (a [simple item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/)) with default configuration. To change default settings, declare an item configuration object. Specify the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) and other properties in it. The example below configures the `HireDate` item:
```

> Use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings (**formData** field names) and objects (item configurations). Use a string to create a label-editor pair (a [simple item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/)) with default configuration. To change default settings, declare an item configuration object. Specify the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) and other properties in it. The example below configures the `HireDate` item:

Reviewer 2: ```suggestion
Use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings (**formData** field names) and objects (item configurations). Use a string to create a label-editor pair (a [simple item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/)) with default configuration. To change the default settings, declare an item configuration object: specify the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) and other properties. The example below configures the `HireDate` item:
```
